<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1355</width>
    <height>821</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <family>Noto Sans</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>ChinaticShark</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Triangular</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextBrowser" name="hexView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>500</y>
      <width>991</width>
      <height>311</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Mono</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QTreeWidget" name="packetList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>991</width>
      <height>441</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="columnCount">
     <number>6</number>
    </property>
    <column>
     <property name="text">
      <string notr="true">1</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string notr="true">2</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string notr="true">3</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string notr="true">4</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string notr="true">5</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string notr="true">6</string>
     </property>
    </column>
   </widget>
   <widget class="QTreeView" name="headerView">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>210</y>
      <width>341</width>
      <height>601</height>
     </rect>
    </property>
    <attribute name="headerVisible">
     <bool>false</bool>
    </attribute>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="devEdit">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>10</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="filterEdit">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>30</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>10</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>device</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>30</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>filter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>10</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QTreeWidget" name="threadList">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>50</y>
      <width>341</width>
      <height>151</height>
     </rect>
    </property>
    <property name="columnCount">
     <number>2</number>
    </property>
    <column>
     <property name="text">
      <string notr="true">1</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string notr="true">2</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="deleteButton">
    <property name="geometry">
     <rect>
      <x>1290</x>
      <y>10</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="packetClearButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="protocolEdit">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>20</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="searchButton">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>10</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>16</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>protocol</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetButton">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>10</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ipEdit1">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>10</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>0</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>ipaddr1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ipEdit2">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>30</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>19</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>ipaddr2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ClearButton">
    <property name="geometry">
     <rect>
      <x>1220</x>
      <y>10</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
